package.json

```diff
@@ -41,7 +43,7 @@
   "author": "",
   "license": "",
   "dependencies": {
-    "3dmol": "1.1.4",
+    "3dmol": "^1.3.7",
```

Comment:
is this on purpose? because this caused some issues when I tried to update it, which is why I removed the "^"

Reply:
The latest version of `3dmol` allows using `import` instead of `require`. But I can skip this update since this is not required at this point.

Reply:
hey if it works it works, I just found that it was breaking the install step on e2e